### PR TITLE
add addInputErrorAndHalt to Command

### PR DESCRIPTION
**Pull request by ArekusuNaito**
https://linear.app/grid-alternatives/issue/GRI-2559/add-addinputerrorandhalt-to-command

### CODE REVIEW

File: src/command.test.js [hunk 1]
```diff
@@ -116,6 +116,46 @@ describe('Command', () => {
...
+            return "if you see this, then this test doesn't work"
+          }
+        }
+        beforeEach(() => {
+          spyOn(InputHaltCommand.prototype, 'operation1')
+          spyOn(InputHaltCommand.prototype, 'operation2')
+          spyOn(InputHaltCommand.prototype, 'operation3')
+        })
+
+        it.only('should halt execution', async () => {
```
Comment (ArekusuNaito) [inline src/command.test.js:151]: Remove this

File: src/command.test.js [hunk 2]
```diff
@@ -116,6 +116,46 @@ describe('Command', () => {
...
+          const outcome = await InputHaltCommand.run()
+          expect(outcome.success).toBe(false)
+          expect(InputHaltCommand.prototype.operation1).toHaveBeenCalled()
+          expect(InputHaltCommand.prototype.operation2).toHaveBeenCalled()
+          expect(InputHaltCommand.prototype.operation3).not.toHaveBeenCalled()
```
Comment (ArekusuNaito) [inline src/command.test.js:156]: From prior art.

File: src/command.test.js [hunk 3]
```diff
@@ -116,6 +116,46 @@ describe('Command', () => {
           expect(outcome.errors.input1[0][0]).toEqual('invalid')
         })
       })
+      describe('.addInputErrorAndHalt()', () => {
+        class InputHaltCommand extends Command {
+          static schema =
+          {
+            coolestInput: { type: 'string' }
+
```
Comment (azimux) [inline src/command.test.js:124]: does the linter run here?

Comment (ArekusuNaito) [reply src/command.test.js]: As we mentioned on huddle. For some reason this is allowed for `standard.js` 🤷‍♂️

File: src/command.test.js (hunk 1 above)
Comment (ArekusuNaito) [reply src/command.test.js]: Addressed.

**Review by azimux — APPROVED** [review]

**Pull request merged**